src/main/java/org/opentripplanner/model/Station.java

```diff
@@ -126,7 +126,7 @@ public String getCode() {
   }
 
   /** Additional information about the station (if needed) */
-  public String getDescription() {
+  public I18NString getDescription() {
```

Comment:
The data fetcher in `StopPlaceType` needs to be updated for this.

Reply:
Done